docs/fidesops/docs/guides/saas_postprocessors.md

```diff
@@ -28,6 +28,32 @@ endpoints:
 
 Note: Order matters as it's defined in the config. In the above example, unwrap will be run first, then the output of unwrap will be used in the filter strategy.
 
+### For Update/Delete/GDPR endpoints
+
+For postprocessors defined in update requests, we're postprocessing the access request rows that were retrieved in a format that's needed for update statement.
```

Comment:
Having a hard time following this. Does this mean: "Post-processors defined as part of an `update` request in your `saas_config` can format retrieved rows into an update statement"? Just let me know if that's not the intent and I can rewrite!

Reply:
This was tricky to word properly. But I wanted it to be clear that we're post-processing the results of the access request, instead of post-processing the results of the erasure request. The post-processed results eventually translate into one or more update/delete statements, depending on how many items are in the results array.

Reply:
Can we simplify? "Post-processors can format the results of your access requests for use in subsequent update or delete statements." Then just move right into the `update` example.

Reply:
Perfect, thanks so much!